README.md

```diff
@@ -3,7 +3,7 @@
 The Vulcan Academy's operating system
 
 1. The name of the game is Vulcanomicon
-1. The goal of the game is to create a learning organization
+1. The goal of the game is to develop an operating system for the Vulcan Academy
```

Comment:
What is meant by 'operating system'?

Reply:
The system of rules governing the operation of the organization. The choice of words is intended to imply that the rules will be automated by software over time.

Reply:
Makes sense 👍